modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1037,15 +1041,34 @@ EFAElement3D::createChild(const std::set<EFAElement *> & CrackTipElements,
           center += cut_plane_points[i];
         center /= cut_plane_points.size();
 
+        for (unsigned int i = 0; i < cut_plane_points.size(); ++i)
+        {
+          int iplus1 = i < cut_plane_points.size() - 1 ? i + 1 : 0;
+          int iplus2 = i + 2;
+          if (iplus2 + 2 - cut_plane_points.size() >= 0)
+            iplus2 = iplus2 + 2 - cut_plane_points.size();
+
+          EFAPoint center(0.0, 0.0, 0.0);
```

Comment:
This variable is shadowing the variable with the same name outside this block. Again, deleting the block resolves this issue.